app/controllers/api/braintree_controller.rb

```diff
@@ -20,12 +20,12 @@ def options
     {
       nonce: params[:payment_method_nonce],
       user: params[:user],
-      amount: params[:amount].to_i
+      amount: params[:amount].to_i,
+      store: Payment
```

Comment:
There's a nested AR model -  `Payment::BraintreeTransaction`. In the wrapping module `Payment` there's class method for persisting transactions: https://github.com/SumOfUs/Champaign/pull/234/files#diff-d78eaa46b7e7446716f9cac4cab14825R7

I thought we should let any user of our wrapper inject any type of store they wish (or none at all). We're good so long as that injected object responds to `write_transaction`